### ISSUE TITLE
QA Ph2_Accessibility: Extended - Missing form label

### ISSUE THREAD

**Issue by Remiqa** [GoogleChrome/ioweb2016#384]
There is a missing form label on the Extended page when using the WAVE extension for Chrome, set to filter WCAG 2.0 AA.

Steps: 
1. Go to https://phase2-dot-io-webapp-staging.appspot.com/io2016/
2. Click on the I/O Extended tab 
3. Enable the WAVE extension
4. View Style errors with the filter set to WCAG 2.0 AA 
5. Note missing form label 

Browser: 
Chrome 48 Mac OS X 10.11 

![gio_384](https://cloud.githubusercontent.com/assets/7585604/13580941/4f1e16fc-e459-11e5-83c1-5de5a52f5ba1.png)
